src/Objs/Redirect/RedirectComponent.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react'
+import {
+  ensureUserIsLoggedIn,
+  isEditorLoggedIn,
+  isUserLoggedIn,
+  load,
+  provideComponent,
+  urlFor,
+} from 'scrivito'
+import { Redirect } from './RedirectObjClass'
+
+provideComponent(Redirect, ({ page }) => {
+  const requireLogin = page.get('requireLogin')
+  const link = page.get('link')
+
+  React.useEffect(() => {
+    if (!link || isEditorLoggedIn()) return
+
+    if (requireLogin && !isUserLoggedIn()) {
+      ensureUserIsLoggedIn()
+      return
+    }
+
+    load(() => urlFor(link)).then((url) => {
+      if (!url) return
+
+      if (link.isExternal() && window.top) {
+        window.top.location.replace(url)
+      } else {
+        window.location.replace(url)
+      }
+    })
+  }, [requireLogin, link])
+
+  if (isEditorLoggedIn()) {
+    return (
+      <div className="container d-flex h-100">
+        <div className="alert alert-warning d-flex m-auto">
+          <i className="bi bi-exclamation-circle bi-2x" aria-hidden="true"></i>
+          <div className="my-auto mx-2">
+            This obj never renders any content but triggers a redirect to the
+            target specified in the page properties.
+          </div>
+        </div>
+      </div>
+    )
+  }
```

Comment:
Sorry for the rollback: But we need the redirect also for editors. Otherwise one will always "land" on the redirect page, if one clicks "My Tynacoon".

So please remove the `isEditorLoggedIn()` specific code from this component.

Reply:
I now took the best of both solutions 🙂